### PR TITLE
made docsearch attribute reflect version of page

### PR DESCRIPTION
**Pull request by thisisobate**
Signed-off-by: thisisobate <obasiuche62@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes
Made docsearch attribute reflect the version of the page

## Verification
Works on local

### CODE REVIEW

File: website/layouts/_default/baseof.html [hunk 1]
```diff
@@ -2,9 +2,10 @@
 <html lang="en">
 
 <head>
+    {{ $version     := index (split .File.Path "/") 0 }}
```
Comment (squat) [inline website/layouts/_default/baseof.html:5]: Tiny nit: why do we need so many spaces here between the `$version` and the `:=`?

**Review by squat — APPROVED** [review]
Overall this change makes sense. One question, will this result in meta-tags with values like `release-0.2` or `v0.2`?

File: website/layouts/_default/baseof.html (hunk 1 above)
Comment (thisisobate) [reply website/layouts/_default/baseof.html:5]: oh...my bad
would fix up

**Comment by thisisobate** [comment]
it will result in meta tags with values v0.2

**Review by squat — APPROVED** [review]

**Comment by squat** [comment]
cool, just tested this and it works perfectly in the preview :)

**Comment by thisisobate** [comment]
Awesome!

**Pull request merged**